Formula/gpsd.rb

```diff
@@ -19,9 +20,22 @@ class Gpsd < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "eeb280e130d077b69e19bcd74ee415c5791d81e21a74d62248077f0937d6d5e9"
   end
 
+  depends_on "asciidoctor" => :build
+  depends_on "python@3.9" => :build
   depends_on "scons" => :build
 
+  uses_from_macos "ncurses"
+
+  resource "gps" do
+    url "https://files.pythonhosted.org/packages/35/73/958b3163573839d7f83cc84199ac1ce7068c3c39dca548fdfa5d37b93392/gps-3.19.tar.gz"
+    sha256 "9e1d2280e4944ddb46cf19772b2768848db91bf8363ad8a72b7996b1c6dd560a"
+  end
```

Comment:
Is this necessary? It is using v3.19 from 2019.
At least part of the python files are available in main source tar/repo https://gitlab.com/gpsd/gpsd/-/tree/master/gps

The issue would be the lack of setup.py and some files are templates like https://gitlab.com/gpsd/gpsd/-/blob/master/gps/__init__.py.in